src/web/screens/clientSideIntegration.tsx

```diff
@@ -47,6 +58,10 @@ function ClientSideIntegration() {
     loadDomainNames();
   }, [loadDomainNames]);
 
+  useEffect(() => {
```

Comment:
Can all of these be combined into one useEffect()?  And better yet, can we use the loader pattern that Lionell established to load all of these when the page itself loads?